### ISSUE TITLE
Github / OAuth2 working but has several bugs

### ISSUE THREAD

**Issue by pavilalopes** [tchiotludo/akhq#1516]
I've successfully got GitHub SSO working, but it's very glitchy:

- Bug 1: even though login succeeds, upon return from the Oauth callback I'm redirected to the login page again. (I can then manually navigate to /ui and am logged in)
- Bug 2: logout doesn't work. I'm redirected to the login page but the JWT cookie is not deleted. I can, again, manually navigate to /ui and see that I'm logged in. I can logout by deleting the cookie in developer tools.

Running version 0.24.0

Relevant configuration bits:
```yaml
micronaut:
  server:
    context-path: "/kafkahq"
    host-resolution:
      host-header: Host
      protocol-header: X-Forwarded-Proto
  security:
    # Use GitHub for SSO
    enabled: true
    oauth2:
      enabled: true
      clients:
        github:
          client-id: "xxx"
          client-secret: "xxx"
          scopes:
            - user:email
            - read:user
          authorization:
            url: https://github.com/login/oauth/authorize
          token:
            url: https://github.com/login/oauth/access_token
            auth-method: client-secret-post

akhq:
  security:
    default-group: no-roles
    oauth2:
      enabled: true
      providers:
        github:
          label: "Login with GitHub"
          username-field: login
          users:
            - username: xxx
              groups: [admin]

  connections:
    # snipped
```

**Comment by AlexisSouquiere** [comment]
I think that the logout issue is not linked to the OAuth2 authentication.
We have the same issue and I figured out that it's due to the context-path in Micronaut settings.

For the 1st issue I fixed it in 0.24.0 but if it's not working on your side, there may be something else. Can you tell me if you have always this behaviour or maybe only after and logout when you try to login again ? This one was the issue I fixed in 0.24.0

**Comment by tchiotludo** [comment]
everything have been rewrite on https://github.com/tchiotludo/akhq/pull/1472, can you try with latest dev version and reopen if it's not working ?
